pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -455,21 +454,16 @@ func (s *DiscoveryServer) WorkloadUpdate(id string, workloadLabels map[string]st
 			//   case 2: the workload xDS connection has not been established,
 			//           also no need to trigger a full push here.
 			if connection.modelNode.IPAddresses[0] == id {
-				fullPush = true
+				// There is a possibility that the pod comes up later than endpoint.
```

Comment:
I am surprised this is possible have you seen it? i guess the pod probably comes up first but pilot gets the endpoint before it?

Reply:
that's what i mean, we can not guarantee when pilot get a pod/endpoint